packages/web-components/src/components/va-accordion/va-accordion.tsx

```diff
@@ -65,8 +65,8 @@ export class VaAccordion {
     let headerText;
     let headerLevel;
     getSlottedNodes(clickedItem, null).map((node: HTMLSlotElement) => {
-      headerText = node.innerHTML;
-      headerLevel = parseInt(node.tagName.toLowerCase().split('')[1]);
+      headerText = node?.innerHTML
+      headerLevel = node?.tagName?.toLowerCase().split('')[1]
```

Comment:
Do we have tests around what happens when these conditionals aren't met?

Reply:
They are optional conditionals so they would just return `undefined` when not met. 
By getting back `undefined` which is falsy that helps determine which value should be used in the analytics event on lines 78 and 80

Reply:
Do we have tests for what happens when we get a falsy value? I just want to make sure that we have tests to prevent a regression in the future, especially since this is a bug fix.

Reply:
Is the `node` going to be undefined here, or is it `innerHTML` and `tagName` that might be undefined?

Reply:
@bkjohnson node in IE11 returns a string of EmptyTextNode as shown in the screenshots and thus has no properties available on it

Reply:
Test code that I used in IE11 which included both the `slot=headline` method and the `header` props method
![Screenshot (29)](https://user-images.githubusercontent.com/11822533/152258999-21786499-7047-481e-99ec-a7a70317b9fc.png)

The `header` props method returns back `EmptyTextNode` for `node` and the `slot=headline` method returns the `<h6>` element from the sample code which is why we need the optional chaining checks
![Screenshot (28)](https://user-images.githubusercontent.com/11822533/152258998-1a28ff72-ef14-4300-8193-aef1800cc35a.png)